### ISSUE TITLE
 ActiveX Control Warning on Windows 10 in Tools

### ISSUE THREAD

**Issue by dkav** [linkagescape/linkage-mapper#12]
On Windows 10 Creators Update, when you select an empty field in a toolbox an ActiveX control warning appears. See this [GeoNet discussion](https://community.esri.com/thread/194209-axtivex-error-warning) for more details. 

A fix will require updating all tool stylesheets. Locally, I have updated LmDlgContent.xsl and the changes seem to have resolved the issue. The fix is based off MdDlgContent.xsl from ArcGIS 10.6.  I do not know if there are implications for other setups.

**Comment by johngallo** [comment]
This relates to the versioning discussion that is underway.

**Comment by dkav** [comment]
Yes and no. I have experienced this problem before in other versions, although I cannot prove it now. Moreover, the stylesheets have been not updated to reflect ESRI's updates since I made the first sheet many years ago and probably should be. Only testing in other environments will validate if it is backward compatible or not.

(From my perspective, the versioning conversation was primarily about ArcGIS rather than Windows, although it doesn't have to. It should include GRASS and Circuitscape though.)
